cmd/oras/pull.go

```diff
@@ -85,6 +85,7 @@ Example - Pull files with local cache:
 }
 
 func runPull(opts pullOptions) error {
+	printed := make(map[string]bool)
```

Comment:
It seems that `printed` is accessed in RW mode concurrently. Should we use `sync.Map` instead?

Reply:
Billy said he discussed with you and you had decided that we do not need `sync.Map` in this case.

Reply:
That's not true. `FindSuccessors` might be invoked concurrently and thus `printed` may be RW concurrently and it causes race condition.

Reply:
I'll change it to sync.Map.